docs/plugins/advanced_topics/npe2_migration_guide.md

```diff
@@ -235,16 +235,16 @@ has to be imported or run until a compatible writer is found.
 
 ```python
 def napari_get_writer(
-    path: str, layer_types: List[str]
+    path: str, layer_types: list[str]
```

Comment:
Same comment as above, maybe we should leave this since it's from an era when List was the correct type hint.